woocommerce/wc-admin
# Adjust the timing of the Marketing.php note to 5 days.

Fixes #5923 

This ensures the Marketing.php note does not show until the store has been active 5 days.

### Testing

1. Set the `woocommerce_admin_install_timestamp` to a timestamp 5 days before today's date
2. Install WP Crontrol plugin, in the plugin's list of cron jobs, click "run now" on the `wc_admin_daily` job
3. Confirm that the marketing note is present in the home screen inbox
